tests/testMatchFits.py

```diff
@@ -44,20 +44,20 @@ def setUp(self):
         self.cat1 = afwTable.SimpleCatalog(self.schema)
         self.cat2 = afwTable.SimpleCatalog(self.schema)
         for i in range(self.size):
-            record1 = self.cat1.table.makeRecord()
-            record2 = self.cat2.table.makeRecord()
+            record1 = self.cat1.makeRecord()
+            record2 = self.cat2.makeRecord()
```

Comment:
I believe this was ancient code from before we could call `makeRecord` directly on catalogs, and I prefer that unit tests demonstrate the preferred way to do things. However @TallJimbo may know better so I'll ask him to weigh in.